### PR TITLE
[T3A5][W09-A4]Muhammad Mustaqiim Bin Muhar

### PR DESCRIPTION
**Pull request by mustaqiimuhar**

### CODE REVIEW

**Comment by mustaqiimuhar** [comment]
Ready for review

File: src/seedu/addressbook/ui/Formatter.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+public class Formatter {
+
+    /** A decorative prefix added to the beginning of lines printed by AddressBook */
+    private static final String LINE_PREFIX = "|| ";
+
+    /** A platform independent line separator. */
+    private static final String LS = System.lineSeparator();
```
Comment (hwkchia) [inline src/seedu/addressbook/ui/Formatter.java:16]: You might want to consider renaming the variable.

File: src/seedu/addressbook/ui/TextUi.java [hunk 2]
```diff
@@ -65,7 +46,7 @@ private boolean shouldIgnore(String rawInputLine) {
      * @return true if input line is a comment.
      */
     private boolean isCommentLine(String rawInputLine) {
-        return rawInputLine.trim().matches(COMMENT_LINE_FORMAT_REGEX);
+        return rawInputLine.trim().matches(formatter.getCommentLineFormatRegex());
```
Comment (hwkchia) [inline src/seedu/addressbook/ui/TextUi.java:49]: This is more pertaining to input formatting.  And you formatter's SR is output formatting only?

**Review by hwkchia — COMMENTED** [review]
Good attempt at applying SRP.  Keep it up!

**Comment by hwkchia** [comment]
ACK by closing PR.

**Pull request closed without merging**